ecosystem/node-checker/src/evaluators/metrics/consensus/round.rs

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use super::{
+    super::{
+        common::{get_metric, GetMetricResult},
+        types::{MetricsEvaluatorError, MetricsEvaluatorInput},
+    },
+    types::CATEGORY,
+};
+use crate::{
+    configuration::EvaluatorArgs,
+    evaluator::{EvaluationResult, Evaluator},
+    evaluators::EvaluatorType,
+};
+use anyhow::Result;
+use clap::Parser;
+use log::debug;
+use poem_openapi::Object as PoemObject;
+use prometheus_parse::Scrape as PrometheusScrape;
+use serde::{Deserialize, Serialize};
+
+// TODO: When we have it, switch to using a crate that unifies metric names.
+// As it is now, this metric name could change and we'd never catch it here
+// at compile time.
+const METRIC: &str = "aptos_consensus_last_committed_round";
+
+#[derive(Clone, Debug, Deserialize, Parser, PoemObject, Serialize)]
+pub struct ConsensusRoundEvaluatorArgs {}
+
+#[allow(dead_code)]
+#[derive(Debug)]
+pub struct ConsensusRoundEvaluator {
+    args: ConsensusRoundEvaluatorArgs,
+}
+
+impl ConsensusRoundEvaluator {
+    pub fn new(args: ConsensusRoundEvaluatorArgs) -> Self {
+        Self { args }
+    }
+
+    fn get_consensus_round(
+        &self,
+        metrics: &PrometheusScrape,
+        metrics_round: &str,
+    ) -> GetMetricResult {
+        let evaluation_on_missing_fn = || {
```

Comment:
nit: you could share this across consensus evaluators (and others) with some utils.